holoviews/plotting/renderer.py

```diff
@@ -433,6 +433,8 @@ def _render_ipywidget(self, plot):
             from IPython.display import display
             display(data, raw=True)
             return {'text/html': '<div style="display: none"></div>'}, {}
+        elif config.comms == 'colab':
+            load_notebook(config.inline)
```

Comment:
This seems to be unrelated to this PR :upside_down_face:

Reply:
Indeed :)